projects/plugins/jetpack/modules/sso/class.jetpack-sso-helpers.php

```diff
@@ -426,6 +426,18 @@ public static function delete_connection_for_user( $user_id ) {
 
 			return $xml->getResponse();
 		}
+
+		/**
+		 * Check if we should load Jetpack_SSO_User_Admin.
+		 *
+		 * @return bool
+		 */
+		public static function should_enable_sso_user_admin() {
+			if ( Jetpack_Options::get_option( 'sso_user_admin_load' ) === 'disabled' ) {
```

Comment:
What do you think about adding a filter here, so the feature can be disabled via filters as well?